README.md

```diff
@@ -44,17 +44,18 @@ Your contributions are essential for making Aidermacs the best AI pair programmi
 ### Sample Config With Straight
 ```emacs-lisp
 (use-package aidermacs
-  :straight (:host github :repo "MatthewZMD/aidermacs" :files ("*.el"))
+  :vc (:url "https://github.com/MatthewZMD/aidermacs" :rev :newest)
```

Comment:
Once [this](https://github.com/melpa/melpa/pull/9373) is merged, we hopefully don't need this line anymore.

Reply:
https://github.com/melpa/melpa/pull/9373